### DISCUSSION TITLE
Command line

### DISCUSSION THREAD

**Discussion by zs-dima** [mikekazakov/nimble-commander#383] [Ideas]
Could be nice to add textbox to enter a terminal command to execute in the current folder.
It would be a toolbar textbox or command line above a status bar.
Don't need to open a terminal for one command, we already have the option to open a terminal if needed something more than one command execution.
Very useful to navigate to a certain folder and just type terminal command + enter to execute it. Quick search has to have a key prefix for this case.

**Comment by mikekazakov** [comment]
NC has already something very similar with the overlapped terminal mode:
Ctrl+Alt+Up => type a command => Enter => command executed.

Otherwise this functionality can be added as an external tool.

**Reply by ABC-INTERACTIVE** [reply]
Wow! Fantastic!

**Reply by zs-dima** [reply]
@mikekazakov thanks a lot.
1. Looks it is Control+Option+Up in the last nightly version.
2. Overlap terminal path doesn't follow active panel path in the last version for some reason.

**Reply by mikekazakov** [reply]
Yeah, Alt == Opt, same thing.
If you have a deterministically reproducible scenario when the overlapped terminal doesn't follow the active panel - please follow [CONTRIBUTING.md](https://github.com/mikekazakov/nimble-commander/blob/main/CONTRIBUTING.md) and submit the issue.
